### DISCUSSION TITLE
Can I use separate remote repositories for "regular" files and LFS?

### DISCUSSION THREAD

**Discussion by the-drunk-coder** [git-lfs/git-lfs#5694] [Q&A]
Hi, 

I wonder whether it's possible to use one repo (on GitHub or GitLab) for the code and one (on my own server) for large (audio) files? 

I have a little VPS and could install something like Soft Serve that I could use for the LFS part. I sense this would be possible but I couldn't piece it fully together so far, so I wonder if someone could give me a hint here.

Do I need to create a repo on the LFS server?

I'd rather use my already existing OwnCloud space, i.e. with local folder storage, but I've read in another thread that this isn't recommendable.

**Comment by bk2204** [comment]
You can use a separate remote for LFS files using the `lfs.url` setting, which can be added in `.lfsconfig` in the repository if you want.  That way you can store the LFS files elsewhere.  The `git-lfs-config(5)` documentation outlines how to set it.

If you want a small Git LFS server, we also offer https://github.com/git-lfs/lfs-test-server/ as an option.  It can be used in production in a number of configurations.

If you were planning on using OwnCloud to mount some cloud data as a cloud syncing service, then no, that wouldn't be recommended for storing repositories since it can very much cause corruption.
